api/accounts/index.ts

```diff
@@ -11,41 +12,43 @@ async function handleAddressRoute(req, res) {
             apiStatusCode: '400',
```

Comment:
change status code to a number rather than a string

Reply:
done